### PR TITLE
install and sample fixes

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: _docs/samples/bookinfo.md [hunk 1]
```diff
@@ -87,14 +87,26 @@ reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
 reviews-v2-1343845940-b34q5                 2/2       Running   0          6m
 reviews-v3-1813607990-8ch52                 2/2       Running   0          6m
 ```
-1. Determine the ingress' URL:
+1. Determine the gateway ingress URL:
+
+  If your cluster is running in an environment that supports external loadbalancers,
+use the ingress' external address:
```
Comment (andraxylia) [inline _docs/samples/bookinfo.md]: This change is not needed. The ADDRESS field in the ingress resource will always be set to the correct value. @ijsnellf could you please confirm?

File: _docs/samples/bookinfo.md [hunk 2]
```diff
@@ -87,14 +87,26 @@ reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
...
 ```bash
 $ kubectl get ingress
 NAME      HOSTS     ADDRESS                 PORTS     AGE
 gateway   *         130.211.10.121          80        1d
-$export GATEWAY_URL=130.211.10.121:80
+$ export GATEWAY_URL=130.211.10.121:80
```
Comment (andraxylia) [inline _docs/samples/bookinfo.md]: Same as above.

File: _docs/samples/bookinfo.md [hunk 3]
```diff
@@ -184,10 +199,10 @@ route:
 
 ### Fault Injection
 
-   To test our bookinfo application microservices for resiliency, we will _inject a 7s delay_
-   between the reviews:v2 and ratings microservices. Since the _reviews:v2_ service has a
-   10s timeout for its calls to the ratings service, we expect the end-to-end flow to
-   continue without any errors.
+To test our bookinfo application microservices for resiliency, we will _inject a 7s delay_
```
Comment (andraxylia) [inline _docs/samples/bookinfo.md]: _inject --> inject
delay_ --> delay

File: _docs/tasks/istio-installation.md [hunk 4]
```diff
@@ -32,15 +32,19 @@ git clone https://github.com/istio/istio
 ```bash
 cd istio
 ```
-2. Install the Istio core components (Istio-Manager, Mixer, and Ingress-Controller):
+1. Install the Istio core components (Istio-Manager, Istio-Mixer, and Ingress-Controller):
```
Comment (andraxylia) [inline _docs/tasks/istio-installation.md]: Please revert to Mixer, as per terminology doc.

File: _docs/tasks/istio-installation.md [hunk 5]
```diff
@@ -32,15 +32,19 @@ git clone https://github.com/istio/istio
...
+or
+```bash
+kubectl apply -f ./kubernetes/istio-16.yaml # for Kubernetes 1.6 or later
 ```
 1. Source the Istio configuration file:
 ```bash
 source istio.VERSION
 ```
-3. Download one of the [`istioctl`]({{site.bareurl}}/docs/reference/istioctl.html) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
+1. Download one of the [`istioctl`]({{site.bareurl}}/docs/reference/istioctl.html) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
```
Comment (andraxylia) [inline _docs/tasks/istio-installation.md]: mtail just committed some changes in install related to site.bareurl and buller points. You can incorporate the same.

File: _docs/samples/bookinfo.md (hunk 1 above)
Comment (frankbu) [inline _docs/samples/bookinfo.md]: On minikube:
```
$ kubectl get ingress
NAME      HOSTS     ADDRESS   PORTS     AGE
gateway   *                   80        3h
```

File: _docs/samples/bookinfo.md (hunk 3 above)
Comment (frankbu) [inline _docs/samples/bookinfo.md]: Why should "delay" not be italics?

File: _docs/tasks/istio-installation.md (hunk 4 above)
Comment (frankbu) [inline _docs/tasks/istio-installation.md]: Done, and merged with mtail's other changes in master

File: _docs/samples/bookinfo.md (hunk 3 above)
Comment (andraxylia) [inline _docs/samples/bookinfo.md]: Ok, I did not realize this is italics, I thought it is a typo.

**Pull request merged**